docs/notebooks/doc2vecc_newsgroup20.ipynb

```diff
@@ -164,47 +349,6 @@
     "    print(\"%s classified:\\t\\t %i correctly,\\t %f%%\" % (k,true_positives, true_positives/len(df_positive_negative) * 100))\n",
     "    print(\"%s average distance: \\t %f\" % (k, avg_distance))\n"
    ]
```

Comment:
Thanks very much for your hard work on this.

Just to make sure I'm following, here you're comparing the results from Chen's C implementation of Doc2VecC with your Gensim implementation, the best results for which were observed with corruption rate = 0.2.  And "classified correctly" means articles from the same news category were closer to one another than to an article from a random different category.  Have I got this right?

If so, is 77.012265% better or worse than the word2vec averaging method you were using as a baseline?

Also, while this method of comparison seems reasonable, it might be worth trying the method Chen uses in the paper to evaluate Doc2VecC for document classification -- training a linear SVM on the Doc2VecC representations of these news articles and then comparing its performance to one trained on the averaged word2vec vectors (or whatever baseline you prefer)

Reply:
It look like this latest notebook *doesn't* compare with Chen's C-patch results – which would still be useful as a sanity check, because i remember seeing in a prior verision of the notebook a comparison where there was a large discrepancy between vectors that should have performed similarly. 

It would be useful for comparisons to also include timings – as I believe the summed-docvec may add a noticeable overhead – and to compare against plain Word2Vec with a larger window (as the effect of using "20% of words from anywhere in doc" instead of just "small context window" is a little bit like "larger context window").

Reply:
It's a bit weird but `Chen's C-patch results` are the in last commit 7 hours ago. I'll list down the timings in a sec.

Reply:
@GeneralSemantics Sorry for late. Yes, that's what correcly classified means. I wanted a quick experiment with a clean dataset that is easy to import. 

SVM on top could be something to try out, I'll reread the paper next week and see if it's manageable, thank you.